LinuxEX1.txt

```diff
@@ -0,0 +1,197 @@
+#firts exercise
+#a
+
+touch nameFile.txt
+cd nameFile.txt
+echo "antonna" > nameFile.txt
+cat nameFile.txt
+
+#b
+mkdir new
+
+#c
+cp ~/nameFile.txt ~/new
+cat new/nameFile.txt
+
+#d
+gedit ~/AntonNahhas.txt
+
+#e
+cd ~/new
+cp ../AntonNahhas.txt newFile.txt
+ls
+cat newFile.txt
+
+#f
+cd ~/new
+ls
+
+#g
+#first way
+cd /home
+#second way
+cd ~/new
+cd ..
+cd ..
```

Comment:
both of the methods you provided are valid ways to go back to the home directory. However, you can simplify it by using cd with the tilde (~) symbol, which represents the home directory.